### ISSUE TITLE
Cannot configure objects if I add additional Backends

### ISSUE THREAD

**Issue by it-native** [sni/Thruk#1067]
**Describe the bug**
If I add one or more backends, the "umbrella" server itself disappears in the Config-Tool

**Thruk Version**
Thruk 2.34-2 / OMD 3.30 

**To Reproduce**
1. Add additional backend(s)
2. Config Tool (-> Objects Settings)
3. First addional Backend is selected in the top header, server itself is gone

**Expected behavior**
All backends show in the config tool & can be selected

**Screenshots**
![image](https://user-images.githubusercontent.com/5801642/103181580-d3306c80-48a2-11eb-9baf-d5a3a17ffe21.png)
![image](https://user-images.githubusercontent.com/5801642/103181599-112d9080-48a3-11eb-9d0a-f4c56658ad74.png)

**Desktop (please complete the following information):**
 - Firefox 83

**Comment by sni** [comment]
this should have been fixed already. Could you by chance try the latest release?

**Comment by it-native** [comment]
> 
> 
> this should have been fixed already. Could you by chance try the latest release?

Sure can do ... all on 4.00 or just the "umbrella"?

**Comment by sni** [comment]
this is a local issue, so updating umbrella is sufficient.

**Comment by it-native** [comment]
thanks - with thruk 2.40-2 on OMD 4.00 the Umbrella works fine.

What's the last thruk/omd (3.x) Version without this issue? I think with OMD 3.20 i didn't have this issue?

**Comment by sni** [comment]
no, idea. But it should be fine with omd 3.40 as well because that contains the same Thruk release.

**Comment by it-native** [comment]
OK, .. little update this:
if i create a new site with 4.00 it works fine, if I update the site from 3.30 to 4.00 the problem still exists. 
Tested with Debian Buster
